src/components/vm/consoles/serial.tsx

```diff
@@ -161,11 +175,27 @@ export const SerialMissing = ({
     );
 };
 
-export const SerialPending = () => {
+export const SerialPending = ({
+    vm,
+} : {
+    vm: VM,
+}) => {
+    const Dialogs = useDialogs();
+
     return (
         <EmptyState icon={PendingIcon} status="custom">
             <EmptyStateBody>
-                {_("Restart this virtual machine to access its serial console")}
+                <EmptyStateBody>
+                    {_("Restart this virtual machine to access its serial console")}
+                </EmptyStateBody>
+                <EmptyStateFooter>
+                    <Button
+                        variant="secondary"
+                        onClick={() => Dialogs.show(<VmRestartDialog vm={vm} />)}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2376-5e904223-20251103-111358-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)